tools/boards.txt.py

```diff
@@ -68,7 +68,7 @@
             'crystalfreq_menu',
             'flashfreq_menu',
             'flashmode_menu',
-            '512K', '1M', '2M', '4M', '8M', '16M',
+            '1M', '2M', '4M', '8M', '16M', '512K',
```

Comment:
I see the 512K moved to then end of lists in other spots, is this on purpose?

Reply:
Yes, so the default flash space for esp8266 generic board is 1MB.
512K boards are obsolete and not sold anymore, also abandonned / not usable in nonos-sdk-v3.
We still have them anyway, but no more as default.